controllers/util.go

```diff
@@ -241,3 +243,16 @@ func NotFound(err error) bool {
 		errType == reflect.TypeOf(users_permissions.NewRemoveUserPermissionNotFound()).String() ||
 		errType == reflect.TypeOf(db.NewGetDBSizeNotFound()).String()
 }
+
+func environmentDisabled(object client.Object) bool {
+	envs := os.Getenv(disabledEnvironments)
```

Comment:
Not a huge fan of reading environment variables in a random util function, but I can live with it and we seem to do similar things already so let's leave it

Reply:
Moved it into the InitEnv() function where it is enabled in main though it is not passed vie reconciliation object. Will keep the same approach as it already was.